source/module_io/to_xml.h

```diff
@@ -0,0 +1,29 @@
+/*
+To print any kind of information to an XML file
```

Comment:
Sorry, I don't understand the usage of this file?

Reply:
Currently in ABACUS there are many output files organized in the way like xml but the root tag is absent.
The present xml-like format will only be quite slightly benificial for organizing human-readable information but for large files, few people will read them, instead, machine/program will be the ultimate one read them and do postprocessing. However, due to ill-defined file structure, the absent root tag will cause failure of xml file parse (like xml library of Python), which means the xml-like format will not be beneficial for both human and program!
Therefore, I want to add this small library uniformly control and standardize the xml-like output, so that parsing some output will be easier.
But I just create this file here and not complete the implementation yet...

Reply:
I have moved this file to another branch~